src/Integration/Binding/CSharpVBBindingOperation.Writer.cs

```diff
@@ -70,11 +75,6 @@ internal partial class CSharpVBBindingOperation
                 return existingRuleSetPath;
             }
 
-            if (ShouldIgnoreConfigureRuleSetValue(currentRuleSetPath))
```

Comment:
The flow before was:
1. Call TryUpdateExistingProjectRuleSet
  1.1 It checks if IsDefaultRuleSet -- returns false
2. We continue and check again IsDefaultRuleSet

Now it's simpler:
1. Check if IsDefaultRuleSet
2. If not - call TryUpdateExistingProjectRuleSet